parsers/IL.py

```diff
@@ -109,14 +106,27 @@ def fetch_production(
         raise NotImplementedError("This parser is not yet able to parse past dates")
 
     data = fetch_all()
-    production = [float(item) for item in data]
+    production = data.get("Production")
+    reserve = data.get("CurrentReserve")
+
+    if not production or not reserve:
+        raise ValueError("Could not parse Noga production")
+
+    date = data.get("PeakReserveDate")
+    time = data.get("PeakReserveTime")
+    date_time = arrow.get(f"{date} {time}", "DD/MM/YYYY HH:mm").datetime
```

Comment:
We should make sure this generates valid datetimes before passing it to the production object.